Docs/source/usage/parameters.rst

```diff
@@ -94,6 +94,11 @@ Overall simulation parameters
     ``self_fields_required_precision``, this parameter may be increased.
     This only applies when warpx.do_electrostatic = labframe.
 
+* ``warpx.self_fields_verbosity`` (`integer`, default: 2)
+    The vebosity used for MLMG solver for space-charge fields calculation. Currently
+    MLMG solver looks for verbosity levels from 0-5. A higher number results in more
+    verbose output. This only applies when warpx.do_electrostatic = labframe.
```

Comment:
Is it really only for `warpx.do_electrostatic = labframe`? You also added the argument to `WarpX::AddSpaceChargeField` so it should also apply to the relativistic functionality, right?